Cesium.CodeGen/Ir/Types/PrimitiveType.cs

```diff
@@ -188,9 +188,9 @@ internal static class PrimitiveTypeInfo
         { PrimitiveTypeKind.SignedChar, (OpCodes.Ldind_I1, OpCodes.Stind_I1) },
         { PrimitiveTypeKind.UnsignedChar, (OpCodes.Ldind_U1, OpCodes.Stind_I1) },
         { PrimitiveTypeKind.Short, (OpCodes.Ldind_I2, OpCodes.Stind_I2) },
-        { PrimitiveTypeKind.UnsignedShort, (OpCodes.Ldind_I2, OpCodes.Stind_I2) },
+        { PrimitiveTypeKind.UnsignedShort, (OpCodes.Ldind_U2, OpCodes.Stind_I2) },
         { PrimitiveTypeKind.Int, (OpCodes.Ldind_I4, OpCodes.Stind_I4) },
-        { PrimitiveTypeKind.UnsignedInt, (OpCodes.Ldind_I4, OpCodes.Stind_I4) },
+        { PrimitiveTypeKind.UnsignedInt, (OpCodes.Ldind_U4, OpCodes.Stind_I4) },
         { PrimitiveTypeKind.Float, (OpCodes.Ldind_R4, OpCodes.Stind_R4) },
         { PrimitiveTypeKind.Long, (OpCodes.Ldind_I8, OpCodes.Stind_I8) },
         { PrimitiveTypeKind.UnsignedLong, (OpCodes.Ldind_I8, OpCodes.Stind_I8) },
```

Comment:
There's `Ldind_U8` as well, can you update this one as well? Other than this, LGTM, thanks for fix and investigation!

Reply:
ldind_u8 does not exist.

Reply:
Ah, I wasn't attentive enough when reading the instruction list. I double-checked it is present in the list, but missed the fact it's declared as an alias for `ldind.i8`.
![image](https://github.com/ForNeVeR/Cesium/assets/92793/bdd7b9ff-c9cd-44a4-b5f5-71d5b1295e94)